component-classes/index.js

```diff
@@ -1,54 +1,52 @@
 export const attention = {
   base: 'border-2 relative',
   tooltip:
-    'i-bg-$color-tooltip-background i-border-$color-tooltip-background i-shadow-$shadow-tooltip i-text-$color-tooltip-text rounded-4 py-6 px-8',
-  callout: 'i-bg-$color-callout-background i-border-$color-callout-border i-text-$color-callout-text py-8 px-16 rounded-8',
-  popover:
-    'i-bg-$color-popover-background i-border-$color-popover-background i-text-$color-popover-paragraph-text rounded-8 p-16 drop-shadow-m',
-  arrowBase:
-    'absolute h-[14px] w-[14px] border-2 border-b-0 border-r-0 rounded-tl-4 transform',
+    'i-bg-$color-tooltip-background i-border-$color-tooltip-background shadow-m s-text-inverted rounded-4 py-6 px-8',
+  callout: 'i-bg-$color-callout-background i-border-$color-callout-border s-text py-8 px-16 rounded-8',
```

Comment:
Do we plan to remove all the `i-` classes in the future? or these are fine for certain use cases?

Reply:
But we should for sure try to get rid of us many as possible by creating adequate semantic tokens. 🙂